docs/source/install/index.rst

```diff
@@ -53,7 +53,7 @@ on Ubuntu, or ``sudo yum install curl nss`` on RHEL/CentOS. Then run this comman
 
 .. code-block:: bash
 
-   curl -sSL https://stackstorm.com/packages/install.sh | bash -s -- --user=st2admin --password='Ch@ngeMe'
+   curl -sSL https://stackstorm.com/packages/install.sh | bash -s -- --user=st2admin --password=Ch@ngeMe
```

Comment:
Not sure how enclosing the password value with single quotes helps, considering it doesn't contain any special characters that might be interpreted in bash.

Any chance there were any other uncommitted changes?

Reply:
This was a mistake and a commit was missing. I confused the configured remotes on my workstation.

Reply:
No worries, I've done the same. 😄